user_tools/src/spark_rapids_tools/cmdli/argprocessor.py

```diff
@@ -278,6 +291,13 @@ def define_invalid_arg_cases(self):
                 [ArgValueCase.VALUE_A, ArgValueCase.VALUE_A, ArgValueCase.IGNORE]
             ]
         }
+        self.rejected['Cluster By Properties Cannot go with OnPrem'] = {
+            'valid': False,
+            'callable': partial(self.validate_onprem_with_cluster_props),
+            'cases': [
+                [ArgValueCase.VALUE_A, ArgValueCase.VALUE_C, ArgValueCase.UNDEFINED]
```

Comment:
I am bit confused on above case - Shouldn't it be `[ArgValueCase.VALUE_A, ArgValueCase.VALUE_B, ArgValueCase.UNDEFINED]` ?

Reply:
Thanks @nartal1. I will change this.